app/addons/documents/header/header.actions.js

```diff
@@ -0,0 +1,62 @@
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+
+define([
+  'app',
+  'api',
+  'addons/documents/header/header.actiontypes'
+],
+function (app, FauxtonAPI, ActionTypes) {
+
+  return {
+    toggleCollapseDocuments: function () {
+      FauxtonAPI.dispatch({
+        type: ActionTypes.COLLAPSE_DOCUMENTS
+      });
+
+      FauxtonAPI.Events.trigger('headerbar:collapse');
```

Comment:
I asked Ben regarding that because I sometimes see that they are added to the constants file and sometimes not:

"Yeah. So here's my rationale: events that are relevant script-wide (stuff in /addons/fauxton) for example should be there. Stuff that's addon-specific shouldn't."